keras_cv/utils/conditional_imports.py

```diff
@@ -17,6 +17,34 @@
 except ImportError:
     waymo_open_dataset = None
 
+try:
+    import cv2
```

Comment:
this still depends on cv2?

Reply:
Yeah, if we are going to refactor it we should rely on PIL AKA Pillow as core keras does.

The good news is this doesn't change the API signature, so we can do it as a community contribution or follow up.